doc/style.css

```diff
@@ -1,109 +0,0 @@
-/* A style for ocamldoc. Daniel C. Buenzli */
```

Comment:
Why?

Reply:
Well, we're going to be generating the docs with odoc so I don't think this style will work anymore. The default odoc style seems to be inspired by dbuenzli's style for ocamldoc, if that's a small consolation.

Reply:
Alright.